docs/resources/privatelink.md

```diff
@@ -3,12 +3,21 @@
 page_title: "risingwavecloud_privatelink Resource - terraform-provider-risingwavecloud"
 subcategory: ""
 description: |-
-  A Private Link connection
+  A Private Link connection on the RisingWave Cloud platform.
+  In AWS, it is a configured endpoint to connect to a VPC endpoint service in your VPC.
+  In GCP, it is a connected project to a service attachment in your private network.
```

Comment:
Where does the term `project` come from, any reference?

From GCP doc, it seems that `endpoint` is a better-fitting term: https://cloud.google.com/vpc/docs/private-service-connect#endpoints

Reply:
Same for the description in `internal/provider/doc.go`